File: src/Server/Server/Core/Constants/Roles.cs
namespace Server.Core.Constants;

public static class Roles
{
    public const string SuperAdmin = "SuperAdmin";
    public const string SalesManager = "SalesManager";
    public const string WarehouseKeeper = "WarehouseKeeper";
    public const string PurchasingManager = "PurchasingManager";

    public static readonly string[] AllAdmins = { SuperAdmin };
    public static readonly string[] AllManagers = { SuperAdmin, SalesManager, WarehouseKeeper, PurchasingManager };
}

File: src/client/src/features/settings/lib/roles.ts
export const roleKeyMap: Record<string, string> = {
  SuperAdmin: "settings.roles.superAdmin",
  SalesManager: "settings.roles.salesManager",
  PurchasingManager: "settings.roles.purchasingManager",
  WarehouseKeeper: "settings.roles.warehouseKeeper",
  HRManager: "settings.roles.hrManager",
};

export const roleOptions: { value: string; labelKey: string }[] = [
  { value: "SuperAdmin", labelKey: "settings.roles.superAdmin" },
  { value: "SalesManager", labelKey: "settings.roles.salesManager" },
  { value: "PurchasingManager", labelKey: "settings.roles.purchasingManager" },
  { value: "WarehouseKeeper", labelKey: "settings.roles.warehouseKeeper" },
  { value: "HRManager", labelKey: "settings.roles.hrManager" },
];